Ilikescience/governance updates (#366)
design-tokens/community-group: governance/DECISION_MAKING.md

## Core Principles

- **Right-sized process**: Match the process weight to the decision impact
- **Transparent by default**: All decisions happen in public spaces
- **Time-bounded**: Every decision has a clear timeline
- **Action-oriented**: Require specific alternatives, not just objections
- **Clear ownership**: Every decision has clear approvers

## Decision Tiers

### Tier 1: Editorial Changes (Immediate)

#### Scope

Typo fixes, grammar corrections, formatting improvements, documentation updates, meeting notes

#### Process

- Any editor can merge immediately
- Create PR for transparency and history
- Use `editorial` GitHub label

#### Examples

Fixing spelling errors, updating formatting for consistency, correcting broken links in documentation

### Tier 2: Standard Changes (3-week review)

#### Scope

Bug fixes, clarifying examples, small non-conflicting additions, updating references, new features, minor breaking changes, new token types

#### Process

> - Create PR with `standard-change` label

Reviewer: Suggestion only:
My suggestion is to avoid using acronyms in order to make the text more accessible cognitive-wise.
```suggestion
- Create a Pull Request with the `standard-change` label
```

> - Post PR link and a brief summary to the DTCG Slack channel and the email list.

Reviewer: suggestion only:
Can we add the link to the DTCG Slack Channel, please? I couldn't find it.

```suggestion
- Post the Pull Request link and a brief summary to the [Design Tokens Community Group Slack channel](LINK TO SLACK CHANNEL HERE) and the email list.
```

- 3-week comment period
- Need approval from 2 editors
- No unresolved objections from editors

#### Examples

Adding HSL examples to complement RGB, clarifying token usage, fixing incorrect code samples, adding new token properties

### Tier 3: Major Decisions (6-week full process)

#### Scope

New specification modules, major breaking changes, architectural decisions, deprecating features, governance changes, ecosystem-wide changes

#### Process

- Create formal proposal in `/proposals` directory
- Add `major-decision` label

> - Include RFC summary (problem, solution, impact, alternatives)

Reviewer: Suggestion only:
```suggestion
- Include the RFC (Request For Comments) summary (problem, solution, impact, alternatives)
```

- Announce the formal proposal in Slack, GitHub Discussions, and the email list (using the RFC summary).
- 4-week RFC period with at least one public meeting
- Author creates response document addressing feedback
- 1-week voting period (need 2/3 majority of editors). The result of the vote and a record of editor attendance/voting will be published per the Communication and Transparency section.
- 1-week final objection period

> #### RFC Summary Format

Reviewer: Suggestion only:
```suggestion
#### RFC (Request For Comments) Summary Format
```
